modules/channel-web/src/views/lite/components/messages/Message.tsx

```diff
@@ -101,7 +102,10 @@ class Message extends Component<Renderer.Message> {
 
   render() {
     const type = this.props.type || (this.props.payload && this.props.payload.type)
+    const wrappedType = this.props.payload.wrapped && this.props.payload.wrapped.type
```

Comment:
Is it possible that payload be undefined? the previous const makes that check, it would be safe to repeat it

Reply:
fixed